lib/surface.ex

```diff
@@ -181,9 +181,14 @@ defmodule Surface do
       |> Keyword.merge(static_props)
       |> rename_id_if_stateless(module.component_type())
 
+    renamed_slot_props =
```

Comment:
To avoid confusion, maybe we should call this `slot_assigns` this way we have a clear separation between slots/props (inputs) and assigns (internal data). After all, we don't actually rename the props, we just map them to assigns. WDYT?

Reply:
I agree.